### ISSUE TITLE
Optimize for angular error when encoding normal maps

### ISSUE THREAD

**Issue by MarkCallow** [BinomialLLC/basis_universal#251]
Please add a new mode to both the etc1s and uastc encoders to optimize the encoded result for angular error rather than PSNR. This can be used when encoding normal maps particularly 2-component normal maps separated into RGB and A.

**Comment by richgel999** [comment]
Yes - this is on our TODO list. (It should boost quality, although I suspect not as much as people think.)
